frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts

```diff
@@ -62,8 +62,7 @@ export const ListPageCreateButton: React.FC<ListPageCreateButtonProps> = require
   .ListPageCreateButton;
 export const ListPageCreateDropdown: React.FC<ListPageCreateDropdownProps> = require('@console/internal/components/factory/ListPage/ListPageCreate')
   .ListPageCreateDropdown;
-export const ListPageBody: React.FC = require('@console/internal/components/factory/ListPage/ListPageBody')
-  .default;
+export { default as ListPageBody } from '@console/dynamic-plugin-sdk/src/app/components/factory/ListPage/ListPageBody';
```

Comment:
Is this needed with the change to `frontend/packages/console-dynamic-plugin-sdk/src/app/components/index.ts`?

Reply:
removed, thank you :slightly_smiling_face: